item-system/demonlord.js

```diff
@@ -0,0 +1,151 @@
+mergeObject(CONFIG, {
+  xacus: {
```

Comment:
```suggestion
  illandril: {
```

To keep things consistent, I'd like all CONFIG settings for this module to live under `CONFIG.illandril.hotbarUses`, that way the instructions can be simply...
> If you are a module author, you can add support right in your own module by adding a regular expression to this module's config (`CONFIG.illandril.hotbarUses.macros.{system}`, where `{system}` is one of the supported systems).

... as opposed to needing to know which person added support for each specific system.

Reply:
Makes absolutely sense.